### ISSUE TITLE
Desktop Notification Improvements

### ISSUE THREAD

**Issue by djeber** [RocketChat/Rocket.Chat#3227]
Your Rocket.Chat version: (30)

Currently the desktop notification feature on Rocket.Chat is very good but it is very basic when you compare it to the one on Facebook for example when you click on it it doesn't disappear while on Facebook when, you click on it it take you to the message and disappear and also on Facebook you get notification even if you are not opening Facebook, it will be great if Rocket.Chat implement those features to its desktop notification

**Comment by brendanheywood** [comment]
Yes agree - when you focus on the chat window which is related to the notification is should auto dismiss the popup. It often obscures the chat window itself and gets in the road:

![image](https://cloud.githubusercontent.com/assets/187449/15884584/f71a52e2-2d94-11e6-93ae-2f9f7b39fed1.png)

It can get really bad when you've had a bunch of notifications and one whole side of your screen can fill up

![image](https://cloud.githubusercontent.com/assets/187449/15884661/5fd6cc3e-2d95-11e6-88dd-27558174c64e.png)

These 'queue' up and in chrome / ubuntu at least it only shows 3 notifications, so if you manually dismiss one then it gets replaced with another one off the queue.
